packages/boxel/addon/components/boxel/apply-changes-toggle/index.css

```diff
@@ -12,8 +12,8 @@
 
 .boxel-apply-changes-toggle--active {
   background-image:
-    url('../@cardstack/boxel/images/icons/arrows.svg'),
-    url('../@cardstack/boxel/images/icons/arrows.svg');
+    url('/@cardstack/boxel/images/icons/arrows.svg'),
+    url('/@cardstack/boxel/images/icons/arrows.svg');
```

Comment:
Is the slash correct here?

Reply:
> Nice changes. I only have the one question around the paths starting with `/`. Is that some sort of webpack trick?

I don't know. It crashes if I remove the slash. It works with it somehow, without doing any relative paths.